src/Microsoft.Azure.WebJobs/ODataFilterResolutionPolicy.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    /// <summary>
+    /// Marker type for Table resolution
+    /// </summary>
+    internal sealed class ODataFilterResolutionPolicy
```

Comment:
>internal [](start = 4, length = 8)

It'd be fine to make this public so that other attributes could use the same OData resolution logic that Table does .